CMakeLists.txt

```diff
@@ -76,6 +76,9 @@ find_package(Lua)
 find_package(LibDataChannel)
 find_package(re)
 
+set(OPENDSS_INCLUDE_DIRS "/usr/local/openDSSC/bin/")
```

Comment:
It would be nice if we can move this into `cmake/FindOpenDSS.cmake`.

Let me know, if I can support you with the CMake stuff.